lib/improver/weighted_blend.py

```diff
@@ -57,20 +252,31 @@ def __init__(self, coord, coord_adjust=None):
     def __repr__(self):
         """Represent the configured plugin instance as a string."""
         return (
-            '<BasicWeightedAverage: coord = {0:s}>').format(self.coord)
+            '<WeightedBlend: coord = {0:s}>').format(self.coord)
 
     def process(self, cube, weights=None):
-        """Calculate weighted mean across the chosen coord
+        """Calculate weighted blend across the chosen coord
 
         Args:
             cube : iris.cube.Cube
                    Cube to blend across the coord.
             weights: Optional list or np.array of weights
-                     or None (equivalent to equal weights)
+                     or None (equivalent to equal weights).
 
         Returns:
             result : iris.cube.Cube
-                     containing the weighted mean across the chosen coord
+                     containing the weighted blend across the chosen coord.
+
+        Raises:
+            ValueError 1 : first argument not a cube.
```

Comment:
If it is in the google style guide it would be helpful if there was an example in the wiki pages on good code practice so we knew how to implement this. I agree that labelling them ValueError 1 etc it is not very help and in the long run is likely to make the code hard to maintain.